docker-compose.yml

```diff
@@ -19,13 +19,13 @@ services:
         RABBITMQ_DEFAULT_USER: 'zulip'
         RABBITMQ_DEFAULT_PASS: 'zulip'
     volumes:
-      - '/opt/docker/zulip/rabbitmq:/var/lib/rabbitmq:rw'
+      - 'rabbitmq-vol:/var/lib/rabbitmq:rw'
   redis:
     image: 'quay.io/sameersbn/redis:latest'
     volumes:
-      - '/opt/docker/zulip/redis:/var/lib/redis:rw'
+      - 'redis-vol:/var/lib/redis:rw'
   zulip:
-    image: 'zulip/docker-zulip:1.9.0'
+    image: 'zulip/docker-zulip'
```

Comment:
This is bad for production, you don't just want your Zulip instance to update when you don't want it to.

Reply:
Alright, will put it back